src/Gui/Navigation/NavigationStyle.h

```diff
@@ -100,12 +100,13 @@ class GuiExport NavigationStyle : public Base::BaseClass
         Clip        = 4,  /**< Clip objects using a lasso. */
     };
 
-    enum OrbitStyle {
-        Turntable,
+    enum OrbitStyle
+    {
+        RoundedArcball,
         Trackball,
-        FreeTurntable,
         TrackballClassic,
-        RoundedArcball,
+        Turntable,
+        FreeTurntable
     };

```

Comment:
Same here.

Though we shouldn't have a copy of the enum. We should use this one in the cpp file instead (if possible)